src/components/proofs/WorkProof.tsx

```diff
@@ -53,7 +54,7 @@ const workTitleLeft = classnames(
 export default function () {
   const [loading, setLoading] = useState(false)
   const [open, setOpen] = useState(true)
-  const [email, setEmail] = useState<string | undefined>()
+  const [email, setEmail] = useState<string | undefined>('@bwl.gg')
```

Comment:
We won't see the `placeholder` then, let's remove it

Reply:
Removed it